x/ecocredit/client/tx.go

```diff
@@ -81,23 +81,27 @@ Parameters:
 
 func txCreateBatch() *cobra.Command {
 	return &cobra.Command{
-		Use:   "create-batch [issuer] [class_id] [start_date] [end_date] [metadata] [issuance]",
+		Use:   "create-batch [issuer] [class_id] [start_date] [end_date] [project_location] [metadata] [issuance]",
```

Comment:
Hmm... this has too many positional args now. We should probably make them all flags